gulpfile.js

```diff
@@ -30,12 +30,14 @@ gulp.task('browserSync', function() {
 })
 
 gulp.task('sass', function() {
-  return gulp.src('app/scss/**/*.scss') // Gets all files ending with .scss in app/scss and children dirs
-    .pipe(sass()) // Passes it through a gulp-sass
-    .pipe(gulp.dest('app/css')) // Outputs it in the css folder
-    .pipe(browserSync.reload({ // Reloading with Browser Sync
-      stream: true
-    }));
+    return gulp.src('app/scss/**/*.scss')  // Gets all files ending with .scss in app/scss and children dirs
+        .pipe(sass({
+            style: 'compressed',
```

Comment:
Please remove style. This is a preference and shouldn't be imposed on others, especially beginners.